packages/services/api/src/modules/schema/providers/registry-checks.ts

```diff
@@ -854,3 +859,19 @@ function isFederationRelatedChange(change: SchemaChangeType) {
 function compareAlphaNumeric(a: string, b: string) {
   return a.localeCompare(b, 'en', { numeric: true });
 }
+
+const dangerousBreaking: Rule = ({ changes }) => {
+  return changes.map(change => {
+    if (change.criticality.level === CriticalityLevel.Dangerous) {
+      return {
+        ...change,
+        criticality: {
+          ...change.criticality,
+          level: CriticalityLevel.Breaking,
+        },
+      };
+    }
+
+    return change;
+  });
+};
```

Comment:
Should we really change the criticality of the schema?

Instead we could also adjust the check whether the schema check is a failure? https://github.com/graphql-hive/console/blob/cbf8b935f0e4b1549f27f414999b82c3c9166cfa/packages/services/api/src/modules/schema/providers/registry-checks.ts#L538-L567

Have you thought about it and wighted the pros and cons of these two different approaches?

Reply:
I was initially thinking it would be very similar. But I do have concerns with my approach in that it will make it hard to distinguish them in the UI.
I like this recommendation, I think it will do exactly what we need while not making the change types as ambiguous.

Reply:
I've adjusted the implementation and terminology in our UI and documentation.
Adjusting the variable names and db column now. But I'm considering this resolved for all intensive purposes.